data/data.py

```diff
@@ -29,8 +29,35 @@ def check_hashes(d):
     print("There are " + str(counter) + " correct files.")
     return all_good
 
+def make_hash_list(directory, title):
```

Comment:
Where are you calling this from? it seems like travis is not covering this function, thats why it is failing.